### ISSUE TITLE
FPS limiter can't be set to 0

### ISSUE THREAD

**Issue by viperguy212** [Kaldaien/FAR#121]
In an attempt to hit 100fps via FAR I've enabled the 60fps option as indicated in the instructions but upon using "Click + enter" to type in a frame limiter value of 0 FAR resets my value to 60fps. 

FPS limiter seems to work properly under 60fps but not a single value over that. I also attempted to set the limiter to various higher values (100, 144, etc.) with no avail. 

Thanks

**Comment by Idearum** [comment]
Click + enter ? What? Do you mean Ctrl+Click ?

**Comment by viperguy212** [comment]
Apologies, I meant that as "Ctrl + click." The value after entering "0"  defaults to "60.000 fps no preference".

**Comment by Idearum** [comment]
Then it /is/ disabled. The "Framerate Limit" checkbox at the right will have been unchecked to indicate that the frame rate limiter is not enabled. The FPS limiter can have three different modes:

* 60.000 fps (No Preference) == No limit engaged, no graphical preference over what the frame rate limiter graph should target. This can only be 60 FPS as it is the default value when "no preference" aka 0 FPS is configured.

* 60.000 fps (Graphing Only) == No limit engaged, custom graphical preference over what the frame rate limiter graph should target. This can be 60 FPS, or whatever else you want.

* 60.000 fps (Limit Engaged) == Frame rate limiter enabled, configured to 60 FPS (or whatever else you set it to).

So basically there's no issue here, you've already disabled the FPS limiter when you set it to 0 FPS. All that happened was that /the graph/ was configured to target "60 FPS", as that's the default value when the user doesn't set the graphical component to anything particular when the limit isn't being engaged.
